modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/AbstractJavaExtensionModelLoader.java

```diff
@@ -97,7 +98,8 @@
                                                                                          new PrivilegedApiValidator(),
                                                                                          new ComponentLocationModelValidator(),
                                                                                          new InjectedFieldsModelValidator(),
-                                                                                         new IgnoredExtensionParameterModelValidator()));
+                                                                                         new IgnoredExtensionParameterModelValidator(),
+                                                                                         new JavaScopeModelValidator()));
```

Comment:
can't this break any currently allowed scope definitions?

Reply:
forget about this, i saw the change in extensions-api now

Reply:
no. Look at ext-api